### ISSUE TITLE
Missing Toolbar Button

### ISSUE THREAD

**Issue by HoudyDoody** [MrOtherGuy/bookmark-batch-updater#9]
There's no button for bookmark-batch-updater in the toolbar. Closing and reopening FF didn't help. FF 109.0.1 64-bit, Windows 10 21H2 Update: KB5020872 12/17/2022

**Comment by MrOtherGuy** [comment]
Did you check inside the extension puzzle-piece button?
The button of this extension goes to there by default.

**Comment by HoudyDoody** [comment]
Yes, I checked there and all the pull-down menus, bookmark menus, the bookmark star, and other places. I also removed the addon, closed FF, reopened it, reinstalled the addon, and checked "everywhere" again. If I remember correctly, I think I may have tried to install it many moons ago and ran into the same problem.  I could really use this. Thousands of bookmarks from a lot of research; I'm a writer, but I worked 35 or so years building systems. I could edit a JSON backup, but I don't want to go that route if I don't have to.

    On Saturday, February 18, 2023 at 03:37:45 AM EST, MrOtherGuy ***@***.***> wrote:  
 
 


Did you check inside the extension puzzle-piece button?
The button of this extension goes to there by default.

**Comment by MrOtherGuy** [comment]
Okay. If you have set Firefox to "never remember history" then Firefox is essentially always in private browsing mode. And as such you must give the extension permission to run in private windows. You can do that from the addons manager.

Another thing you can check is that maybe the button is in some other toolbar, for example bookmarks toolbar or menubar both of which may be hidden depending on your settings.

If neither of the above help then you might want to try resetting toolbar settings to default - I'm pretty sure cuatomize mode view has button for that. That will probably move the button back into the extensions-button panel (assuming you have given the extension a permission to run in private windows).
